bbl_screen-patch/patches/printerui/qml/settings/X1PlusOTADialog.qml

```diff
@@ -11,7 +11,7 @@ Item {
     property alias name: textConfirm.objectName
     property var currentVersion: screenSaver.cfwVersions['cfw']['version']
     property var ota: X1Plus.OTA.status()
-    property var otaEnabled: !!X1Plus.Settings.get("ota.enabled", false)
+    property var otaEnabled: !!X1Plus.Settings.get("ota.enabled", True)
```

Comment:
Pretty sure we need this lowercase, similar to false. @jwise can confirm, but lots of languages are very picky on boolean value spelling.